pandapipes/converter/stanet/table_creation.py

```diff
@@ -574,27 +578,26 @@ def create_geodata_sections(row):
     )
 
 
-def create_heat_exchangers(net, stored_data, connection_table, index_mapping, add_layers):
+def create_heat_exchangers(net, stored_data, index_mapping, add_layers, add_flow=False):
```

Comment:
Wouldn't it be better to choose a different name to avoid confusion with create_heate_xchangers from create.py?
e.g. create_heat_exchangers_stanet

Reply:
Not a bad idea, I did rename it.